### DISCUSSION TITLE
Dark mode

### DISCUSSION THREAD

**Discussion by ghost** [standardnotes/forum#2632] [Suggestions] [answered]
Dark mode is very common on Mac and Windows now. Most website offers dark mode using CSS  prefers-color-scheme. Please consider add this feature? 

Even if I enable extensions to theme the interface, I have to go through the white page before logging in...

**Comment by julianfairfax** [comment]
While it's possible to theme Standard Notes with a dark mode using a third-party extension, Standard Notes doesn't follow your system theme if you're using this method and it would be nice to have an official dark mode like on the mobile app. Or at least an option to have Standard Notes follow the system theme using a third-party extension of the user's choice for the dark mode.

**Comment by JaspalSuri** [comment]
Hi @julianfairfax, we plan to add a free dark theme to the [web](https://app.standardnotes.org) and desktop apps by version 4 and plan to add the ability for certain themes to activate in response to the system's light/dark modes at some point after that. (The mobile apps currently support this, though. Simply press and hold a theme and select which mode it should activate in.)

**Comment by pietryszak** [comment]
> 

What is the status of dark mode for desktop app in linux ? How long we need to wait for v4 and dark mode ?

**Comment by ghost** [comment]
any updates on this?

**Comment by atmoio** [comment, marked as answer]
This is available now:

<img width="537" alt="Screen Shot 2022-03-24 at 7 40 46 AM" src="https://user-images.githubusercontent.com/3277844/159918248-8ae674e3-1bc6-4ce7-9edf-9809f0001cde.png">

Note however that the app does not come with a dark theme out of the box. You can either get those themes from our paid plans, or some of the free community themes lying around on the web.

**Comment by julianfairfax** [comment]
> Hi @julianfairfax, we plan to add a free dark theme to the [web](https://app.standardnotes.org) and desktop apps by version 4 and plan to add the ability for certain themes to activate in response to the system's light/dark modes at some point after that. (The mobile apps currently support this, though. Simply press and hold a theme and select which mode it should activate in.)

When is version 4 slated to come into existence then? It's been almost a year and Standard Notes still looks pretty out of place on my laptop :(

**Comment by stefan-yas** [comment]
@julianfairfax

> When is version 4 slated to come into existence then? It's been almost a year and Standard Notes still looks pretty out of place on my laptop :(

What issues you seem to be experiencing?

**Comment by julianfairfax** [comment]
> @julianfairfax
> 
> > When is version 4 slated to come into existence then? It's been almost a year and Standard Notes still looks pretty out of place on my laptop :(
> 
> What issues you seem to be experiencing?

I mean in the way of the theme. Everything is dark and Standard Notes is all light

**Comment by stefan-yas** [comment]
@julianfairfax Are you on a paid plan? If yes, you should be able to enable automatic light/dark themes switching in **Settings -> Appearance ->Use system color scheme**.

**Comment by julianfairfax** [comment]
> @julianfairfax Are you on a paid plan? If yes, you should be able to enable automatic light/dark themes switching in **Settings -> Appearance ->Use system color scheme**.

I'm not so I'm waiting for version 4.0. I just don't know when that'll be released? The mobile app has a dark mode which looks great.

**Comment by ezxpro** [comment]
> @julianfairfax Are you on a paid plan? If yes, you should be able to enable automatic light/dark themes switching in **Settings -> Appearance ->Use system color scheme**.

You have to be a paying user. Here my system is dark, but standard notes is all white.

Btw, I'm a notion paying user, and honestly, this app is pricey in comparison.

**Comment by ghost** [comment]
@julianfairfax 
check out joplin its got everything and dark mode for free.
i didn't want to wait for dark mode
